packages/sdk/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "pipcook-sdk",
+  "version": "1.0.0",
+  "description": "",
+  "main": "dist/index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
```

Comment:
You need to add the test command here.

Reply:
These test cases have to run by workflow, because it depends daemon, so I removed this script.